web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -326,15 +400,20 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
             csrf
           );
           const data =
-            res &&
-            res.payload &&
-            res.payload.data &&
-            res.payload.status === 200 &&
-            !res.error
+            res?.payload?.data && res?.payload?.status === 200 && !res?.error
               ? res.payload.data
               : null;
           if (data) {
             reviews[period.id] = data[0];
+          } else {
+            console.log(res?.error);
```

Comment:
Nice! If you also want to keep the console output, I would change it to use `console.error` instead of `console.log`. That will make it appear in red.